recipes-core/images/intel-aero-image.bb

```diff
@@ -35,6 +35,7 @@ IMAGE_INSTALL += "rng-tools"
 
 # Build tools
 IMAGE_INSTALL += "packagegroup-core-buildessential"
+IMAGE_INSTALL += "catkin"
```

Comment:
@lucasdemarchi I tried this on master-next branch

Image size Before: 1397751808
Image size After  :  1434451968

Just with above change few dependencies are missing, So i created another patch,
https://gist.github.com/avinash-palleti/a1c5fbca55e8f0213ee87d6caa62f14b

This has changes in meta-ros and meta-intel-aero. With these changes i am able to initialize workspace but catkin_make is still failing with following reason,

CMake Error at CMakeLists.txt:59:
find_package(catkin) failed. catkin was neither found in the workspace nor in the CMAKE_PREFIX_PATH.

I think still there are some missing dependencies.